packages/hw-wallets/src/trezor/trezorConnect.ts

```
@@ -0,0 +1,29 @@
+import type { TrezorConnect as TrezorConnectType } from "@trezor/connect-web";
+
+const getTrezorConnect = async () => {
+  if (chrome && chrome.runtime && chrome.runtime.getPlatformInfo) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)